src/solvers/dgmulti/flux_differencing.jl

```diff
@@ -362,6 +369,30 @@ function entropy_projection!(cache, u, mesh::DGMultiMesh, equations, dg::DGMulti
   return nothing
 end
 
+# TODO: REFACTOR. redundant with entropy_projection!
```

Comment:
As commented, we can make a separate PR to refactor? Since this may be a breaking change.

Reply:
Sounds good to me. Thanks!